### ISSUE TITLE
[Hint] Open Graph Protocol

### ISSUE THREAD

**Issue by Malvoz** [webhintio/hint#3118]
I think https://github.com/webhintio/webhint.io/issues/516 warrants a hint for [OGP](https://ogp.me) implementations. 😂

I can update the list of requirements for the respective social platforms (as described in https://github.com/webhintio/webhint.io/issues/516#issue-360078914).

The hint would presumably look for:

- `<meta>` elements starting with `property="og:{...}"`
- and `prefix="og: http://ogp.me/ns#"` (I think recommendation for the prefix is to be specified in the `<html>` or `<head>` elements.)

If the above exist but doesn't fulfill the minimum requirements (e.g. must include an image of at least 200x200 px, must have `property="og:title" etc.`) then the check fail.

Also [Twitter Cards](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started#twitter-cards-and-open-graph)?

**Comment by molant** [comment]
@Malvoz yes please, update the list of requirements. Looks like there should be different errors levels depending on what's missing/wrong (error, warning, info, hint). If you have already an idea of what should be each thing please share it.

Thanks!

**Comment by Malvoz** [comment]
Sorry for the delay, for a proper overview of what is required/recommended for developers based on the support across platforms, I created an [OGP support & requirements](https://github.com/Malvoz/ogp-support-and-requirements) document. But things like formats and correct structure of metadata is explained in https://ogp.me.
